unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/UIBackground/UIBackgroundHandler.cs

```diff
@@ -44,6 +44,7 @@ public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, PBUiB
             image.Slices = model.GetBorder().ToUnityBorder();
             image.UVs = model.Uvs.ToDCLUVs();
             image.ScaleMode = model.TextureMode.ToDCLImageScaleMode();
+            // image.canvas.pickingMode = model.PointerBlocker ? PickingMode.Position : PickingMode.Ignore;
```

Comment:
better to just remove the line

Reply:
Oh yes I intended to do that but forgot, thanks!